ansible/roles/kubernetes-addons/tasks/cluster-monitoring.yml

```diff
@@ -13,6 +13,7 @@
     url=https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/cluster/addons/cluster-monitoring/influxdb/{{ item }}
     dest="{{ local_temp_addon_dir }}/cluster-monitoring/{{ item }}.j2"
     force=yes
+    validate_certs=False
```

Comment:
The problem is that the run fails without this. When I see that other similar code has this set, it seems reasonable.